woocommerce/woocommerce-free-shipping-progress-bar-block
# Add unit tests

Fixes #2 

### Notes

Currently, the components are not covered by unit tests. The only way to find out if a future development breaks existing functionality, is to manually test the functionality.

This PR adds unit tests for the `<ProgressBar>` and `<ProgressLabel>` components.

### Steps to test

1. Check out this PR.
2. Run `nmp ci`.
3. Run `npm test`.
4. Verify that unit tests have successfully been executed.

### Additional notes

Besides the command `npm test`, I also introduced the following commands:
- `npm run test:coverage` → Run tests and show coverage report.
- `npm run test:help` → Show help for the unit tests.
- `npm run test:watch` → Run tests and watch for changes.
